jobs/e2e_node/arm/image-config-serial.yaml

```diff
@@ -0,0 +1,10 @@
+# To copy an image between projects:
+# `gcloud compute --project <to-project> disks create <image name> --image=https://www.googleapis.com/compute/v1/projects/<from-project>/global/images/<image-name>`
+# `gcloud compute --project <to-project> images create <image-name> --source-disk=<image-name>`
+images:
+  ubuntu:
+    image: ubuntu-gke-2204-1-24-arm64-v20230217
```

Comment:
I suggest that we use `image-family` instead, otherwise we'd need to update this job constantly.

Reply:
Done. Updated.